defaults/main.yml

```diff
@@ -33,6 +47,19 @@ OPENCRAFT_ENV_TOKENS:
   SECRET_KEY: 'your-random-secret-key'
   WATCH_ORGANIZATION: 'your-github-organization'
 
+# These env tokes are generated from other ansible variables, entries in
+# this dictionary can be overriden by OPENCRAFT_ENV_TOKENS
+OPENCRAFT_GENERATED_ENV_TOKENS:
```

Comment:
The idea is that `OPENCRAFT_ENV_TOKENS` are manually set (and not generated from configuration), and `OPENCRAFT_GENERATED_ENV_TOKENS` are generated from the rest of configuration. 

OPENCRAFT_ENV_TOKENS variable is overriden for each deployment, and if I'd put things generated from the rest of configuration, when deploying you'd need to copy these automatically generated variables. 

There is no way to tell Ansible: "Dear Ansible I need a hash object but with these additional variables." other than what I did. Please let me know if this explanation is solid :)